nucypher/blockchain/eth/networks.py

```diff
@@ -24,24 +24,28 @@ class NetworksInventory:  # TODO: See #1564
     ORYX = 'oryx'
 
     # TODO: Use naming scheme to preserve multiple compatibility with multiple deployments to a single network?
-    POLYGON = 'polygon'
-    MUMBAI = 'mumbai'
+    POLYGON = 'polygon-mainnet'
+    MUMBAI = 'polygon-mumbai'
```

Comment:
I like the names, but we need to be careful here - changing these values has side effects. The payment network value is stored in Ursula configuration files, which would already exist for someone running v6.0.0 (i.e. that configuration file would have `polygon` as the value). 

If we want to change these values, I think you'll need to incremenent the default configuration version and create a configuration migration script for converting the old version of the configuration file to this version of the configuration file that uses these values - for example, https://github.com/nucypher/nucypher/commit/82fb99022f8d43b6dbb8c0b7e888bb5049d6d194. (cc @KPrasch ).

Reply:
You can also accept both values I guess as a workaround

Reply:
ok cool, i think i'll remove this change and treat that as a separate issue/PR #2918

Reply:
👌  - nice!